packages/api/cms-api/src/dam/files/files.controller.ts

```diff
@@ -38,7 +39,7 @@ export class FilesController {
     ) {}
 
     @Post("upload")
-    @UseInterceptors(DamUploadFileInterceptor(FilesService.UPLOAD_FIELD))
+    @UseInterceptors(DamUploadFileInterceptor(FilesService.UPLOAD_FIELD), FileSanitizationInterceptor)
```

Comment:
nice use of the interceptor 👍 . Is the order here important? Because this is relevant for the public upload as well